inference-engine/src/vpu/graph_transformer/src/stages/mtcnn.cpp

```diff
@@ -112,15 +112,10 @@ ie::CNNNetwork loadSubNetwork(
     // Load network
     //
 
+    // TODO: replace with ICore interface
+    InferenceEngine::Core reader;
     auto binFileName = fileNameNoExt(fileName) + ".bin";
-
-    IE_SUPPRESS_DEPRECATED_START
-    ie::CNNNetReader networkReader;
-    networkReader.ReadNetwork(fileName);
-    networkReader.ReadWeights(binFileName);
-
-    auto network = networkReader.getNetwork();
-    IE_SUPPRESS_DEPRECATED_END
+    auto network = reader.ReadNetwork(fileName, binFileName);
```

Comment:
As I understand, you can call method without second argument, so there is no more need in `binFileName`

Reply:
fixed